src/status_im2/contexts/communities/menus/request_to_join/view.cljs

```diff
@@ -23,7 +23,9 @@
            can-join?
            can-request-access?
            requested-to-join-at]}]
-  (let [agreed-to-rules? (reagent/atom false)]
+  (let [agreed-to-rules? (reagent/atom false)
+        pending?         (rf/sub [:communities/my-pending-request-to-join id])
+       ]
```

Comment:
something minor the bracket can be at the end of the previous line. I dont think `make lint-fix` will help in this case

Reply:
Will adjust!